### ISSUE TITLE
Save player attributes, skills and class to server

### ISSUE THREAD

**Issue by davidcernat** [TES3MP/TES3MP#41]
Currently, none of these things are being saved to the server except on initial character generation.

**Comment by davidcernat** [comment]
With 7691bcc958a1f7360b9c133f0a3e450988c30fce and matching changes in Lua scripts, attributes and skills are now saved to the server whenever they change.

**Comment by Koncord** [comment]
This task intersects with #4

**Comment by Koncord** [comment]
Anyway, I want help you:

``` cpp
void setCustomClass(ESM::Class &klass)
{
    klass.mData.mIsPlayable = 0x1;

/* 
    The following fields are should to be initialized:

    klass.mName;
    klass.mDescription;
    klass.mData.mSpecialization

    std::vector<int> favoriteAttributes;
    klass.mData.mAttribute[0] = favoriteAttributes[0];
    klass.mData.mAttribute[1] = favoriteAttributes[1];

    klass.mData.mSkills[i][0]
    klass.mData.mSkills[i][1]
*/

    MWBase::Environment::get().getMechanicsManager()->setPlayerClass(klass);
    MWBase::Environment::get().getWindowManager()->setPlayerClass(klass);
}

void setDefaultClass(const std::string &classId)
{
    if (!classId.empty())
        MWBase::Environment::get().getMechanicsManager()->setPlayerClass(classId);

    const ESM::Class *klass = MWBase::Environment::get().getWorld()->getStore().get<ESM::Class>().find(classId);
    if (klass)
        MWBase::Environment::get().getWindowManager()->setPlayerClass(klass);
}

int getMajorSkillId(int i)
{
        MWBase::World *world = MWBase::Environment::get().getWorld();
        const ESM::NPC *cpl = world->getPlayerPtr().get<ESM::NPC>()->mBase;
        const ESM::Class *cls = world->getStore().get<ESM::Class>().find(cpl->mClass);

        return cls->mData.mSkills[i][1];
}

int getMinorSkillId(int i)
{
        MWBase::World *world = MWBase::Environment::get().getWorld();
        const ESM::NPC *cpl = world->getPlayerPtr().get<ESM::NPC>()->mBase;
        const ESM::Class *cls = world->getStore().get<ESM::Class>().find(cpl->mClass);

        return cls->mData.mSkills[i][0];
}


ESM::Skill::sSkillNameIds[getSkillId(i, false)].c_str();
ESM::Skill::sSkillNameIds[getSkillId(i, true)].c_str();
```

**Comment by Koncord** [comment]
Resolved by 11eae277a9e44debf7e4c50130c70d30edc38e62
